components/swap-widget/SwapTsAndCs.tsx

```diff
@@ -2,16 +2,21 @@ import { Box, Checkbox, Flex, Show, Text } from "@chakra-ui/react";
 import LinearButton from "components/linear-button";
 import React, { type Dispatch, type SetStateAction, useState } from "react";
 import TermsAndConditionsContent from "../terms-and-conditions/TermsAndConditionsContent";
+import { useWindowManager } from "components/window-manager";
```

Comment:
@arlindlinum I meant you can define in ts config a path to root level, and then just use @/app/window-manager , instead of using two dots (..) this makes component easier to handle because you can move it around folders and imports will still work